pkg/ingester/ingester.go

```diff
@@ -821,6 +829,31 @@ func (i *Ingester) updateLoop(ctx context.Context) error {
 	}
 }
 
+func (i *Ingester) updateUserTSDBConfigs() {
+	for _, userID := range i.getTSDBUsers() {
+		userDB := i.getTSDB(userID)
+		if userDB == nil {
+			continue
+		}
+
+		cfg := &config.Config{
+			StorageConfig: config.StorageConfig{
+				ExemplarsConfig: &config.ExemplarsConfig{
+					MaxExemplars: int64(i.limits.MaxExemplars(userID)),
+				},
+			},
+		}
+
+		level.Info(logutil.WithUserID(userID, i.logger)).Log("meg", "updating max exemplars configuration.")
+		// This method currently updates the MaxExemplars and OutOfOrderTimeWindow. Invoking this method
+		// with a 0 value of OutOfOrderTimeWindow simply updates Max Exemplars.
+		err := userDB.db.ApplyConfig(cfg)
+		if err != nil {
+			level.Error(logutil.WithUserID(userID, i.logger)).Log("msg", "failed to update max exemplars configuration.")
```

Comment:
This one is okay to keep, but let's not be specific about `max exemplars`.

Reply:
Thanks for catching this one.